recipes/tuiview/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python setup.py install --prefix=$PREFIX
```

Comment:
Just to be on the safe side can you change this to:

```shell
$PYTHON setup.py install
```

PS: Not sure you really need the `$PREFIX`